engine/apps/api/views/paging.py

```diff
@@ -20,30 +20,24 @@ class DirectPagingAPIView(APIView):
 
     def post(self, request):
         organization = request.auth.organization
-        from_user = request.user
 
         serializer = DirectPagingSerializer(
             data=request.data, context={"organization": organization, "request": request}
         )
         serializer.is_valid(raise_exception=True)
 
-        users = [(user["instance"], user["important"]) for user in serializer.validated_data["users"]]
-        schedules = [
-            (schedule["instance"], schedule["important"]) for schedule in serializer.validated_data["schedules"]
-        ]
         try:
             alert_group = direct_paging(
                 organization=organization,
-                team=serializer.validated_data["team"],
-                from_user=from_user,
-                title=serializer.validated_data["title"],
+                from_user=request.user,
                 message=serializer.validated_data["message"],
-                users=users,
-                schedules=schedules,
-                escalation_chain=serializer.validated_data["escalation_chain"],
+                team=serializer.validated_data["team"],
+                users=[(user["instance"], user["important"]) for user in serializer.validated_data["users"]],
                 alert_group=serializer.validated_data["alert_group"],
             )
         except DirectPagingAlertGroupResolvedError:
             raise BadRequest(detail=DirectPagingAlertGroupResolvedError.DETAIL)
+        except DirectPagingUserTeamValidationError:
+            raise BadRequest(detail=DirectPagingUserTeamValidationError.DETAIL)
```

Comment:
validates that the user specifies _alteast_ a team _or_ one user (can specify both of course, but both can't be null)